using-mobile-extensions/places/places-usage-reference.md

```diff
@@ -0,0 +1,336 @@
+<b>Signature:</b>
+```
+static func getNearbyPointsOfInterest(forLocation location: CLLocation,
+                                      withLimit limit: UInt,
+                                      closure: @escaping ([PointOfInterest], PlacesQueryResponseCode) -> Void)
+```
+<b>Example Usage:</b>
+```
+let location = CLLocation(latitude: 40.4350229, longitude: -111.8918356)
+Places.getNearbyPointsOfInterest(forLocation: location, withLimit: 10) { (nearbyPois, responseCode) in    
+    print("responseCode: \(responseCode.rawValue) - nearbyPois: \(nearbyPois)")
+}
+```
+{% endtab %}
+{% tab title="Objective-c" %}
+<b>Signature:</b>
+```
++ (void) getNearbyPointsOfInterest: (nonnull CLLocation*) currentLocation
+                             limit: (NSUInteger) limit
+                          callback: ^ (NSArray<AEPPlacesPoi*>* _Nonnull, AEPPlacesQueryResponseCode) closure;
+```
+<b>Example Usage:</b>
+```
+CLLocation *location = [[CLLocation alloc] initWithLatitude:40.4350229 longitude:-111.8918356];
+
+[AEPMobilePlaces getNearbyPointsOfInterest:location
+                                     limit:10
+                                  callback:^(NSArray<AEPPlacesPoi *> *pois, AEPPlacesQueryResponseCode responseCode) {
+    NSLog(@"responseCode: %ld", (long)responseCode);
+    NSLog(@"nearbyPois: %@", pois);
+}];
+```
+{% endtab %}
+{% endtabs %}
+
+<hr />
+
+### processRegionEvent
+
+Passes a `CLRegion` and a `PlacesRegionEvent` to be processed by the Places extension.
+
+Calling this method will result in an Event being dispatched to the SDK's EventHub. This enables rule processing based on the triggering region event.
+
+{% tabs %}
+{% tab title="Swift" %}
+<b>Signature:</b>
+```
+static func processRegionEvent(_ regionEvent: PlacesRegionEvent,
+                               forRegion region: CLRegion)
+```
+<b>Example Usage:</b>
+```
+let region = CLCircularRegion(center: CLLocationCoordinate2D(latitude: 40.3886845, longitude: -111.8284979),
+                              radius: 100,
+                              identifier: "877677e4-3004-46dd-a8b1-a609bd65a428")
+
+Places.processRegionEvent(.entry, forRegion: region)
+```
+{% endtab %}
+{% tab title="Objective-c" %}
+<b>Signature:</b>
+```
++ (void) processRegionEvent: (AEPRegionEventType) eventType
+                  forRegion: (nonnull CLRegion*) region;
+```
+<b>Example Usage:</b>
+```
+CLCircularRegion *region = [[CLCircularRegion alloc] initWithCenter:CLLocationCoordinate2DMake(40.3886845, -111.8284979)
+                                                             radius:100
+                                                         identifier:@"877677e4-3004-46dd-a8b1-a609bd65a428"];
+
+[AEPMobilePlaces processRegionEvent:AEPPlacesRegionEventEntry forRegion:region];
+```
+{% endtab %}
+{% endtabs %}
+
+<hr />
+
+### registerExtension
+
+This API no longer exists in `AEPPlaces`. Instead, the extension should be registered by calling the `registerExtensions` API in the `MobileCore`.
```

Comment:
"This method no long exists in `AEPPlaces`. The extension should be registered by calling the `registerExtensions` method in `MobileCore`."

Additionally, instead of listing it here, I'd link out to the MobileCore documentation, to avoid duplication of data, and to make it easier to keep the docs content up to date.